rocAL/include/loaders/circular_buffer.h

```diff
@@ -73,8 +73,8 @@ class CircularBuffer {
     bool full();
     bool empty();
     size_t _buff_depth;
-    decoded_image_info _last_image_info;
-    std::queue<decoded_image_info> _circ_image_info;    //!< Stores the loaded images names, decoded_width and decoded_height(data is stored in the _circ_buff)
+    DecodedDataInfo _last_data_info;
+    std::queue<DecodedDataInfo> _circ_data_info;    //!< Stores the loaded data names, decoded_width and decoded_height(data is stored in the _circ_buff)
     crop_image_info _last_crop_image_info;              // for Random BBox crop coordinates
     std::queue<crop_image_info> _circ_crop_image_info;  //!< Stores the crop coordinates of the images for random bbox crop (data is stored in the _circ_buff)
```

Comment:
what is actual_host_buffers in line #95 and why is it a vector of vector of chars. If it is just buffer pointers, please declare as uint8_t * pointers.

Reply:
Removed this variable as its unused